CHANGELOG.md

```diff
@@ -1,5 +1,103 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-beta.2
+
+<!-- generated comparing v5.0.0-beta.1..next -->
+
+_Jul 26, 2021_
+
+Big thanks to the 20 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ We introduced new codemod for converting JSS styles to emotion (#27292) @siriwatknp
+  It should help adoption of v5, by making possible the removal of JSS sooner.
+
+- 🐛 The majority of other changes in this release were bug fixes, test utilities and docs.
+
+### `@material-ui/core@5.0.0-beta.2`
+
+- <!-- 54 --> [Autocomplete] Explain how the loading prop works (#27416) @michaldudak
+- <!-- 49 --> [Autocomplete] Update input value when the input changes (#27313) @turtleseason
+- <!-- 09 --> [Autocomplete] Popper is not closing when the Autocomplete is disabled (#27312) @Goodiec
+- <!-- 42 --> [Checkbox] Skip default hover styles with `disableRipple` (#27314) @faan234
+- <!-- 50 --> [Dialog] Fix override paper styles (#27423) @newsiberian
+- <!-- 17 --> [Grid] Remove width prop for rowSpacing (#27326) @sashkopavlenko
+- <!-- 33 --> [Input] Merge `componentsProps` correctly (#27371) @mnajdova
+- <!-- 55 --> [Pagination] Fixed usePagination requires @emotion in development mode (#27348) @ruppysuppy
+- <!-- 07 --> [Pagination] Fix :hover effect on previous/next button (#27304) @Aubrey-Li
+- <!-- 03 --> [Popper] Consistent timing of popper instance creation (#27233) @eps1lon
+- <!-- 45 --> [Select] Add `SelectChangeEvent` for accurate types for event in onChange prop (#27370) @eps1lon
+- <!-- 18 --> [Tabs] Use theme transition duration for the Tab animation (#27303) @florianbepunkt
+- <!-- 35 --> [TextField] Allow custom colors in FormLabel (#27337) @oliviertassinari
+- <!-- 14 --> [TextField] Fix name of componentsProps (#27338) @oliviertassinari
+- <!-- 04 --> [transitions] Make sure inline styles used for transition values if declared (#27140) @eps1lon
+
+### `@material-ui/codemod@5.0.0-beta.2`
+
+- <!-- 57 --> [codemod] Add `optimal-imports` for v5 (#27404) @mnajdova
+- <!-- 48 --> [codemod] Add jss to emotion codemod (#27292) @siriwatknp
+- <!-- 34 --> [codemod] Fix running codemod CLI on Windows (#27395) @michaldudak
+- <!-- 32 --> [codemod] Fix published version (#27384) @eps1lon
+- <!-- 10 --> [codemod] Improve README.md (#27257) @mnajdova
+
+### `@material-ui/unstyled@5.0.0-alpha.41`
+
+- <!-- 36 --> [NoSsr] Move NoSsr to the Unstyled package (#27356) @michaldudak
+
+### `@material-ui/utils@5.0.0-beta.1`
+
+- <!-- 43 --> [utils] Convert createChainedFunction to TypeScript (#27386) @eps1lon
+
+### `@material-ui/system@5.0.0-beta.2`
+
+- <!-- 39 --> [system] Compute display name of `styled` component if `name` isn't set (#27401) @eps1lon
+- <!-- 08 --> [system] Adds missing type for `shouldForwardProp` (#27310) @KLubin1
+
+### `@material-ui/lab@5.0.0-alpha.41`
+
+- <!-- 16 --> [pickers] Use utils for default min and max dates (#27297) @eps1lon
```

Comment:
This can be removed in favor of https://github.com/mui-org/material-ui/pull/27392. #27297 is just an implementation detail.